zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -3580,6 +3626,83 @@ components:
                 description: A list of resource IDs the permission relates to.
                 items:
                   type: string
+    AuthorizationSearchQueryRequest:
+      allOf:
+        - $ref: "#/components/schemas/SearchQueryRequest"
+      type: object
+      properties:
+        filter:
+          allOf:
+            - $ref: "#/components/schemas/AuthorizationFilterRequest"
+    AuthorizationFilterRequest:
+      type: object
+      properties:
+        ownerType:
+          enum:
+            - USER
+            - ROLE
+            - GROUP
+            - MAPPING
+            - UNSPECIFIED
+        ownerKey:
+          type: integer
+          format: int64
+    AuthorizationResponse:
+      type: "object"
+      properties:
+        ownerKey:
+            type: integer
+            format: int64
+        ownerType:
+          enum:
+            - USER
+            - ROLE
+            - GROUP
+            - MAPPING
+            - UNSPECIFIED
```

Comment:
🔧 Let's extract the enum so we don't repeat ourselves in the request and response. I am also rather sure that we will need this enum in other entities soon 🙂